graphula-core/test/README.lhs

```diff
@@ -15,23 +16,27 @@ Graphula is a simple interface for generating persistent data and linking its de
 {-# LANGUAGE DerivingStrategies #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE FlexibleInstances #-}
 module Main where
 
-import Data.Aeson
-import Control.Monad.Trans.Reader (ReaderT)
-import Control.Monad.Logger (NoLoggingT)
-import Control.Monad.Trans.Resource (ResourceT)
-import Data.Typeable
+import Control.Monad (replicateM_)
 import Control.Monad.IO.Class
 import Control.Monad.IO.Unlift
-import Graphula
-import GHC.Generics (Generic)
-import Test.QuickCheck
-import Test.Hspec
-import Database.Persist (Entity(..))
+import Control.Monad.Logger (NoLoggingT)
+import Control.Monad.Trans.Reader (ReaderT)
+import Control.Monad.Trans.Resource (ResourceT)
+import Data.Aeson
 import Database.Persist.Arbitrary ()
+import Database.Persist (Entity(..))
 import Database.Persist.Sqlite
 import Database.Persist.TH
+import Data.Typeable
+import GHC.Generics (Generic)
+import Graphula
+import Graphula.UUIDKey
+import Test.Hspec
+import Test.QuickCheck hiding (suchThat)
```

Comment:
@eborden can you think of a better name than `suchThat`? Trying to avoid a name clash. Other options:

* `where_` (clashes with `Esqueleto` tho)
* `satisfying`
* `given`
* `iff`

Reply:
I vote for `satisfying`. I might toss in `ensuring` too, or `passing`, or `filtered`. `scoped`?

Reply:
Hmm, I think I like `ensure` or `expect` since we're using non-participle `edit`

Reply:
I like `ensure`

![image](https://user-images.githubusercontent.com/545655/74175218-f99dd880-4bfa-11ea-9491-41501a06bcbb.png)